sdk/src/settings/mod.rs

```diff
@@ -160,69 +173,145 @@ impl SettingsValidate for Trust {
     }
 }
 
-// TODO: all of these settings aren't implemented
-// Settings for core C2PA-RS functionality
+/// Settings to configure core features.
+#[cfg_attr(feature = "json_schema", derive(schemars::JsonSchema))]
 #[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
-#[allow(unused)]
-pub(crate) struct Core {
-    debug: bool,
-    hash_alg: String,
-    #[serde(skip_serializing_if = "Option::is_none")]
-    soft_hash_alg: Option<String>,
-    salt_jumbf_boxes: bool,
-    prefer_box_hash: bool,
-    pub(crate) merkle_tree_chunk_size_in_kb: Option<usize>,
-    pub(crate) merkle_tree_max_proofs: usize,
-    compress_manifests: bool,
```

Comment:
what happened to compress_manifests? Are we assuming it is always enabled?

Reply:
The setting was unused and I don't think we have it merged yet to main, so we can add that later.